custom_components/deltadore_tydom/tydom/MessageHandler.py

```diff
@@ -298,7 +298,7 @@ async def get_device(
                 return TydomSmoke(
                     tydom_client, uid, device_id, name, last_usage, endpoint, device_metadata[uid], data
                 )
-            case "boiler" | "sh_hvac" | "electric" | "aeraulic":
+            case "boiler" | "sh_hvac" | "electric" | "aeraulic" | "sh_hvac":
```

Comment:
Just notice it was already listed :wink:

```suggestion
            case "boiler" | "sh_hvac" | "electric" | "aeraulic":
```

Reply:
yes I saw that after... I need sleep